### ISSUE TITLE
Consume bus mapping in state circuit

### ISSUE THREAD

**Issue by therealyingtong** [appliedzkp/zkevm-circuits#47]
We should update #23 to consume the bus mapping output.

Edit: we can do this in a separate PR after #23 merges.

**Comment by miha-stopar** [comment]
Bus-mapping integrated in the first version of state circuit: ab0b56ca75fdc936ed438b25354a3a08f7b14412.
